### DISCUSSION TITLE
🛫 Release 1.2.0

### DISCUSSION THREAD

**Discussion by valnoxy** [valnoxy/t480-oc#9] [Announcements]
Hey 👋,
I hope you all have a great day!

# Changes
The following macOS versions/variants have been added:
   - macOS 13 Ventura (beta)

<hr /><em>This discussion was created from the release <a href='https://github.com/valnoxy/t480-oc/releases/tag/v1.2.0'>🛫 Release 1.2.0</a>.</em>

**Comment by zeeall** [comment]
As far for the installation - you nailed it! I didn't even need an Ethernet, wifi from my Pixel picked up instantly!
